Content.Server/Headset/HeadsetComponent.cs

```diff
@@ -63,6 +63,18 @@ public void Receive(string message, RadioChannelPrototype channel, EntityUid sou
                 MessageWrap = Loc.GetString("chat-radio-message-wrap", ("color", channel.Color), ("channel", $"\\[{channel.LocalizedName}\\]"), ("name", _entMan.GetComponent<MetaDataComponent>(source).EntityName))
             };
 
+            // If the source isn't a player, we set the chat message accordingly
+            if (!_entMan.TryGetComponent(source, out ActorComponent? _))
+            {
+                msg = new MsgChatMessage
+                {
+                    Channel = ChatChannel.Radio,
+                    Message = message,
+                    //Square brackets are added here to avoid issues with escaping
+                    MessageWrap = Loc.GetString("announcement-radio-message-wrap", ("color", channel.Color), ("channel", $"\\[{channel.LocalizedName}\\]"))
+                };
+            }
```

Comment:
This is inherently going to break with NPCs.

Reply:
Do we have NPCs already? I can add a check to make sure they aren't an NPC either

Reply:
Alternatively, what is a check I could do that won't break with NPCs?

Reply:
Have the caller handle it, or alternatively add a method for it if we expect many more.

Reply:
Added changes